undebt/cmd/main.py

```diff
@@ -57,21 +57,21 @@ def _write_result_text(result_text, path, dry_run):
 def _handle_arguments():
     parser = argparse.ArgumentParser(prog='undebt')
     parser.add_argument(
-        '--input', '-i', metavar='path', action='append',
-        help='paths to files or directories (searched recursively for extension) to be modified (if not passed uses stdin)')
+        'paths', nargs='*', metavar='PATH',
+        help='paths to files or directories (searches for extension recursively) to be modified; '
+        'uses stdin if not passed')
     parser.add_argument(
-        '--pattern', '-p', metavar='path', action='append', required=True,
+        '--pattern', '-p', metavar='PATH', action='append', required=True,
         help='paths to pattern definition files')
     parser.add_argument(
-        '--extension',
-        '-e',
-        metavar='ext',
-        action='append',
+        '--extension', '-e', metavar='EXT', action='append',
         help='extensions of files to be modified when searching a directory')
     parser.add_argument(
-        '--multiprocess', '-m', metavar='processes', type=int, default=16,
+        '--multiprocess', '-m', metavar='INTEGER', type=int, default=16,
```

Comment:
traditionally this is `-j`, `--jobs` (make, etc.) or `-P`, `--processes` (`xargs`).

Saying something like `--multiprocess=3` doesn't really read logically.

Also I'm not sure this tool should even have this option, `xargs -P` is going to do better most of the time.